gcp/modules/gpii-flowmanager/resources/flowmanager-couchdb-network-policy.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: flowmanager-couchdb-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: couchdb
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 10.0.0.0/24
+    - podSelector:
+        matchLabels:
+          app: couchdb
+    ports:
+    - protocol: TCP
+      port: 4369
```

Comment:
It looks like `couchdb-network-policy` already has this rule? Am I misunderstanding how this works?

Reply:
This rule looks like it's redundant. I'll remove it.